catalog/validator/catalog-validator-metacardframecenter/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <reference id="wktValidator" interface="org.codice.ddf.validator.wkt.WktValidator"/>
+
+    <bean id="metacardFrameCenterValidator" class="org.codice.ddf.validator.metacard.framecenter.MetacardFrameCenterValidator">
```

Comment:
✏️ Instead of creating this new class, you should be able to do this solely in this blueprint with...
```xml
<bean id="metacardFrameCenterValidator" class="org.codice.ddf.validator.metacard.wkt.MetacardWktValidator">
    <argument ref="wktValidator"/>
    <argument ref="media.frame-center"/>
</bean>
```
(You could probably use the const here too, but I'm not sure.)

Reply:
@coyotesqrl: From what I can tell from the Blueprint spec and from research online, static fields cannot be referenced from blueprint. The best workaround I could find was https://stackoverflow.com/questions/25613479/osgi-aries-blueprint-static-field. In my opinion, using a factory method like that is worse than what I currently have. I **could** just reference the string values for those constants, but that breaks the linkage.

Based on all that it seems like I should just stick with the way it is, but what do you think?

Reply:
I'm on the fence. We've got two new classes being added to the system for little value, but hard-coding the strings does have major drawbacks. And adding a bunch of methods to the constants class just to work around blueprint's shortcoming seems like overkill of a different sort.

I suppose that leaving them as they are is acceptable. It's certainly not _wrong_.